### ISSUE TITLE
Disabling a raised accent button doesn't show a disabled state

### ISSUE THREAD

**Issue by romannurik** [google/material-design-lite#907]
The following code doesn't seem to actually render the button in a disabled visual state:
```
<button class="mdl-button mdl-js-button mdl-button--raised mdl-button--accent mdl-js-ripple-effect" disabled>Test button</button>
```

Screenshot:

![image](https://cloud.githubusercontent.com/assets/100155/8583856/faed1228-259e-11e5-9ee4-dc72565bc9a3.png)

**Comment by Garbee** [comment]
Looks like we may have a specificity problem from ordering...

**Comment by Garbee** [comment]
ah, we actually missed disabling a colored button. Good find! Submitting a PR to fix this.

**Comment by addyosmani** [comment]
It looks like @Garbee's fix should address this. Thanks for raising it, @romannurik!

**Comment by Garbee** [comment]
Fixed with #908 merged
